tools/linter/yamllint/.yamllint

```diff
@@ -1,5 +1,9 @@
 ---
 
+ignore: |
+  # Remove when https://github.com/kubernetes-sigs/gateway-api/issues/1600 is fixed.
```

Comment:
would be great if you can also create another issue in this repo which will revert this code once above issue is fixed

Reply:
https://github.com/envoyproxy/gateway/issues/810